testgrid/config.yaml

```diff
@@ -3218,6 +3239,18 @@ dashboards:
   - name: OpenStack, v1.11 (dev)
     description: Runs conformance tests using kubetest against kubernetes from the release-1.11 branch with cloud-provider-openstack
     test_group_name: ci-periodic-cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.11
+  - name: vSphere, master (dev)
+    description: Runs conformance tests using kubetest against latest kubernetes master with cloud-provider-vsphere
+    test_group_name: ci-periodic-cloud-provider-vsphere-test-e2e-conformance
```

Comment:
if you set up alert, you'll need some email address to receive the alert :-)

ref https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L6636-L6637

Reply:
@krzyzacy thx, i will add email

Reply:
Hi @figo,

Please rename the names to `vSphere - Cloud-Provider` or some variation. `vSphere` by itself will refer to in-tree components. OpenStack uses it to refer to their cloud-provider, so they've never had both cases. We will.

Reply:
@akutz done, thx